aten/src/ATen/native/vulkan/ops/Add.cpp

```diff
@@ -0,0 +1,133 @@
+#include <ATen/native/vulkan/ops/Common.h>
+#include <torch/library.h>
+
+namespace at {
+namespace native {
+namespace vulkan {
+namespace ops {
+namespace {
+
+Tensor add_scalar(
+    const Tensor& self_,
```

Comment:
Nit: usually I saw in pytorch code base that member fields are named with underscore at the end

Reply:
No that's a good idea.  I'll follow up with the same convention.  I'll use self_arg, etc.  I am not really sue whether we use camelCase or snake_case in PyTorch.  It seems to be a mix.  I can find instances of both even under native.